### ISSUE TITLE
Type Error when updating to v1.11.1

### ISSUE THREAD

**Issue by rharriso** [alchemyplatform/alchemy-web3#81]
There seems to be some issue with the types exports for `@alch/alchemy-web3` v1.1.11.

```ts
import {
  AssetTransfersCategory,
  AssetTransfersParams,
  AssetTransfersResult,
} from '@alch/alchemy-web3'

/// Module '"@alch/alchemy-web3"' has no exported member 'AssetTransfersResult'.ts(2305)
```

I wasn't able to see anything strange in the build or structure of the types file between 1.1.9 (working) and 1.1.11.

**Comment by dlazares** [comment]
I'm also not seeing AssetTransfersCategory exported. My autocomplete generates this import 
`import {AssetTransfersCategory} from "@alch/alchemy-web3/dist/esm/alchemy-apis/types";` which results in this error

```
SyntaxError: Unexpected token 'export'
    at Object.compileFunction (node:vm:352:18)
    at wrapSafe (node:internal/modules/cjs/loader:1031:15)
    at Module._compile (node:internal/modules/cjs/loader:1065:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
```

I recently updated from 1.1.9 => 1.1.12

**Comment by dphilipson** [comment]
Sorry for the trouble on this. I'm working on a fix.

**Comment by dphilipson** [comment]
Fix is published in v1.2.1. Really sorry for the inconvenience!

**Comment by dlazares** [comment]
Thanks for hopping on this so fast. You made my day :)
